docs/tldr/pages/linux/tmt.md

```diff
@@ -0,0 +1,36 @@
+# tmt
+
+> Test Management Tool for creating, running and debugging tests.
+> More information: <https://tmt.readthedocs.io>.
+
+- List available tests, plans, or stories:
+
+`tmt {test|plan|story} ls [<pattern>]`
+
+- Run all test steps for each plan:
+
+`tmt run`
+
+- Run selected plans and tests:
+
+`tmt run plan -n <plan_name> test -n <test_name>`
+
+- Show what tests a specific plan would run:
+
+`tmt run -vv discover plan -n <plan_name>`
+
+- Show test results from the last run in a web browser:
+
+`tmt run -l report --how html --open`
```

Comment:
How about documenting first:
- `tmt run <discover|provision|prepare|...>`: as run only the step
- `tmt run -a <discover|provision|prepare|...>`: as run all original steps and override
- Example1: `tmt run -v discover`: Show all tests that would run